scripts/e2e/deploy-merlin.sh

```diff
@@ -28,7 +28,7 @@ install_merlin() {
     --set deployment.image.registry=${DOCKER_REGISTRY} \
     --set deployment.image.repository=merlin \
     --set deployment.image.tag=${VERSION} \
-    --set transformer.image=${DOCKER_REGISTRY}/merlin-transformer:${VERSION} \
+    --set rendered.overrides.StandardTransformerConfig.ImageName=${DOCKER_REGISTRY}/merlin-transformer:${VERSION} \
```

Comment:
Previously the e2e tests weren't even correctly using the temporarily built transformer image that's pushed to the local image registry just before the tests were run. They were instead using an already released version of the image that's much older (0.35.0) https://github.com/caraml-dev/merlin/actions/runs/11853812453/job/33034971181#step:12:2147. This fix ensures that only the locally built and pushed transformer image is used https://github.com/caraml-dev/merlin/actions/runs/11866485521/job/33073372549#step:12:2111